src/components/Camera/index.js

```diff
@@ -90,6 +90,7 @@ export class CameraPure extends React.Component<CameraPureType> {
             className={style.video}
             audio={!!video}
             height={720}
+            facingMode={'user'}
```

Comment:
Will this return the front facing camera also for document autocapture? 
On another note, on FF this syntax will allow the user to select the facing mode on gUM. If we want to force the user facing mode we should probably use `{facingMode: {exact: 'user'}}` but we should check if it's supported on all browsers

Reply:
`exact` will break on desktop I believe, because the facingMode is not defined for most webcams.
The assumption is that if a device only has one camera then it will give whatever camera.

I'm not sure if we should concern ourselves a lot with auto capture to be honest. but happy to discuss it.

Reply:
I think, if we stop being concerned about autocapture then we should remove the feature altogether because adding regressions doesn't seem ideal given that there might be clients using it.

Reply:
actually auto capture is used on desktop only, no? If so then we want it to be user facing.

Reply:
I agree we should remove it. question is if it's part of the scope or not. As you pointed out is matter of it being a regression or not, which it might not be.

Reply:
It should be desktop only but I've just seen it on mobile, which is a regression! And I've managed to reproduce the camera error after permissions granted on Samsung S8.